### ISSUE TITLE
Package wishlist: MakeMKV

### ISSUE THREAD

**Issue by BasixKOR** [rpmfusion-infra/fedy#109]
** Please describe why this package is not eligible for Fedora ?**
Proprietary software.

** Is this software redistributable ?**
The developers signaled positive feedback for having a third-party package: https://forum.makemkv.com/forum/viewtopic.php?p=7549#p7549

Also there are Flatpak releases already available, so I think it is in a safe zone. 

** Is this software an alternate version of a Fedora provided package ?**
N/A.

**Additional context**

**Comment by kwizart** [comment]
Which feature is in the proprietary makemkv that is not in the FLOSS counterpart mkvtoolnix-gui ?
(not a mkv editor myself).

**Comment by kwizart** [comment]
side note: everyone is free to pick this task if interested it.
See also https://rpmfusion.org/Contributors

**Comment by BasixKOR** [comment]
Unlike the name suggests, MakeMKV is a Blu-Ray and DVD ripping software. Do note that the legal status of the software may be controversial and may not be legal depending on the legal jurisdiction.

**Comment by kwizart** [comment]
right, so there is handbrake for this (packaged from rpmfusion-free).

**Comment by Xmetalfanx** [comment]
I agree that their is a legal reason to use this app on your own collection .... the questionable use of it (in some people/organization's) eyes ... makes it a question if it should be included.   I mean maybe a "we better not take chances and not put it in" as a possible stance.    just my 2 cents ... and more of an open question than a statement (what i intended, i mean)
